dbcon/mysql/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ else ()
     add_dependencies(ha_columnstore loggingcpp)
     add_definitions(-DMYSQL_DYNAMIC_PLUGIN -DPLUGIN_COLUMNSTORE_VERSION="${PACKAGE_VERSION}")
     
-    target_link_libraries(ha_columnstore ${ENGINE_LDFLAGS} ${ENGINE_WRITE_LIBS} ${NETSNMP_LIBRARIES} ${SERVER_BUILD_DIR}/libservices/libmysqlservices.a threadpool)
+    target_link_libraries(ha_columnstore ${ENGINE_LDFLAGS} ${ENGINE_WRITE_LIBS} ${NETSNMP_LIBRARIES} ${SERVER_BUILD_DIR}/libservices/libmysqlservices.a threadpool funcexp)
```

Comment:
I am wondering why is funcexp lib linked to ha_columnstore?

Reply:
removed

Reply:
I don't see this change yet.